src/pydrex/pathlines.py

```diff
@@ -64,10 +64,11 @@ def _max_strain(
         _ivp_func,
         [0, -100e6 * 365.25 * 8.64e4],
         point,
-        method="RK45",  # TODO: Compare to LSODA?
-        # first_step=1e10,
+        method="LSODA",
+        first_step=1e10,
         max_step=np.inf,
         # max_step=1e6,
+        t_eval=None,
         events=[_max_strain],
         args=(interp_velocity, interp_velocity_gradient, min_coords, max_coords),
         dense_output=True,
```

Comment:
Same idea here with the tolerances?

Reply:
A few lines below, atol and rtol are set. Maybe those values can be changed, or that could be in another PR after I check LSODA vs RK45 for both pathline and CPO integrators.